File: Feb_6th,7th_Tasks/Java programs/ServerDowntime.java
public class ServerDowntime {
    public static void main(String[] args) {
        int[] status = {1, 1, 0, 0, 0, 1, 0, 0};

        int totalDowntime = 0;
        int longest = 0, current = 0;

        for (int s : status) {
            if (s == 0) {
                totalDowntime++;
                current++;
                longest = Math.max(longest, current);
            } else {
                current = 0;
            }
        }

        System.out.println("Total Downtime Hours: " + totalDowntime);
        System.out.println("Longest Continuous Downtime: " + longest);
    }
}

File: Feb_6th,7th_Tasks/Python programs/ServerDowntime.py
server_status = [1, 1, 0, 0, 0, 1, 0, 0]

downtime = server_status.count(0)

longest = 0
current = 0
for status in server_status:
    if status == 0:
        current += 1
        longest = max(longest, current)
    else:
        current = 0

print("Total Downtime Hours:", downtime)
print("Longest Continuous Downtime:", longest)
